tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -0,0 +1,123 @@
+| Instruction                   | Description                                                                                          |
++===============================+======================================================================================================+
+| **entry_symbol**              | Name of the entry function for initializing the GDExtension. This function should be defined in      |
+|                               | the ``register_types.cpp`` file. Adding this is necessary for the extension to work.                 |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **compatibility_minimum**     | Minimum compatible version. This prevents older versions of Godot from loading extensions that       |
+|                               | depend on features from newer versions of Godot. **Only supported for Godot 4.1+**                   |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **compatibility_maximum**     | Maximum compatible version. This prevents newer versions of Godot from loading the extension.        |
+|                               | **Only supported for Godot 4.3+**                                                                    |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **reloadable**                | Reloads the extension upon recompilation. Reloading is not supported for every exposed class.        |
+|                               | This flag should be mainly used for developing or debugging an extension.                            |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **android_aar_plugin**        | The GDExtension is part of a :ref:`v2 Android plugin <doc_android_plugin>`. During export this flag  |
+|                               | will indicate to the editor that the GDExtension native shared libraries are exported by the Android |
+|                               | plugin AAR binaries.                                                                                 |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Libraries section
+-----------------
+
+In this section you can set the paths to the compiled binaries of your GDExtension libraries.
+By setting feature flags you can filter which version should be loaded and exported with your
+game depending on which feature flags are active. Every feature flag must match to Godot's
+feature flags. For instance ``macos.debug`` means that it will be loaded if Godot has both the
+``macos`` and ``debug`` flag active. Each line of the section is evaluated from top to bottom.
```

Comment:
Does this not support custom feature flags on export? This and below implies only the official ones are accepted